tally_ho/apps/tally/views/reports/turnout_reports_by_admin_areas.py

```diff
@@ -168,28 +175,30 @@ def get_initial_queryset(self):
                 race=F('center__resultform__ballot__electrol_race_id')
                 ).values('id').annotate(
                 races=ArrayAgg('race', distinct=True),
-                number=F('station_number'),
-                num_registrants=F('registrants')
                 )
             voters = 0
             stations_processed = 0
             registrants_in_processed_stations = 0
             for station in station_ids_by_race:
+                station_obj = stations_by_id.get(station.get('id'))
                 # Calculate stations processed and total registrants
                 form_states = get_result_forms_for_station_in_admin_area(
                     tally_id,
-                    admin_level, area_code, station
+                    admin_level,
+                    area_code,
+                    station,
+                    station_obj
                     )
 
                 if form_states.count() == 1 and \
                         form_states[0] == FormState.ARCHIVED:
                     stations_processed += 1
                     registrants_in_processed_stations += \
-                        station.get('num_registrants')
+                        station_obj.registrants
```

Comment:
Could raise `AttributeError` if `station_obj` is `None`

Reply:
Agreed, so I have made sure we reuse the tally stations queryset used when creating stations_by_id that's used when defining station_obj. This will ensure we never encounter such an error